tests/test_scsynth.py

```diff
@@ -0,0 +1,67 @@
+import os
+import pathlib
+import stat
+from tempfile import NamedTemporaryFile, TemporaryDirectory
+import pytest
+from supriya import scsynth
+
+
+@pytest.fixture
+def mock_env_scsynth_path(monkeypatch):
+    monkeypatch.delenv("SCSYNTH_PATH", raising=False)
+
+
+def test_find_argument(mock_env_scsynth_path):
+
+    with NamedTemporaryFile() as tmp:
+        expected = pathlib.Path(tmp.name).absolute()
+        expected.chmod(expected.stat().st_mode | stat.S_IEXEC)
+        got = scsynth.find(expected)
+        assert got == expected
+
+
+def test_find_env_var(mock_env_scsynth_path):
+
+    with NamedTemporaryFile() as tmp:
+        expected = pathlib.Path(tmp.name).absolute()
+        expected.chmod(expected.stat().st_mode | stat.S_IEXEC)
+        os.environ["SCSYNTH_PATH"] = str(expected)
+        got = scsynth.find()
+        assert got == expected
+
+
+def test_find_on_path(mock_env_scsynth_path):
+
+    with TemporaryDirectory() as tmp_dir:
+
+        scsynth_path = pathlib.Path(tmp_dir) / "scsynth"
+
+        with open(scsynth_path, "w"):
+            scsynth_path.chmod(scsynth_path.stat().st_mode | stat.S_IEXEC)
+            os.environ["PATH"] += os.pathsep + tmp_dir
+            got = scsynth.find()
+            expected = scsynth_path.absolute()
+            assert got == expected
+
+
+def test_find_from_fallback_paths(mock_env_scsynth_path, mocker):
```

Comment:
Make sure to null-out `PATH` in this test, otherwise it will fail on systems where `scsynth` is available on the path already.

Reply:
ahh good catch!